integration/solana/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Integration tests with Solang and Solana",
   "scripts": {
     "test": "tsc; ts-node setup.ts; mocha --parallel *.spec.ts",
-    "build": "solang compile *.sol --target solana -v --log-runtime-errors --math-overflow"
+    "build": "solang compile *.sol --target solana -v --math-overflow"
```

Comment:
I feel like we should get rid of the math overflow flag. It should be checked arithmetic everywhere, except inside `unchecked {}` blocks?

However this would fit better into another PR

Reply:
We have an issue open for that #1216